packages/react-components/src/components/ChatMessage/ChatMessageComponent.tsx

```diff
@@ -52,6 +52,11 @@ type ChatMessageComponentProps = {
    * @param userId - user Id
    */
   onRenderAvatar?: OnRenderAvatarCallback;
+  /**
+   * Optional function to provide customized date format.
+   *
+   */
+  customizedFormatTimestampForChatMessage?: (messageDate: Date) => string;
```

Comment:
should we just have 

dateTimeFormat as the name of the prop that is optional?
If its set we use it and if we don't have it or its invalid we use the default behavior?